tests/e2e/matmul/CMakeLists.txt

```diff
@@ -175,812 +61,73 @@ iree_generated_e2e_runner_test(
     "--iree-preprocessing-pass-pipeline=builtin.module\(util.func\(iree-preprocessing-transpose-matmul-pass{input=lhs}\)\)"
   LABELS
     "requires-arm-sme"
-  TARGET_CPU_FEATURES_VARIANTS
-    "arm_64:sme:+sve,+sme"
-)
-
-iree_generated_e2e_runner_test(
```

Comment:
This looks like a lot of dropped tests. Are these just a result of combining small and large?

Reply:
Yes. Those Bazel nested-list-comprehensions are a concise way of generating many tests. In the generated CMakeLists, it's all expanded.